routes/favicon.coffee

```diff
@@ -0,0 +1,27 @@
+request = require 'request'
+fs = require 'fs'
+tmpdir = require 'os'
+	.tmpdir()
+path = require 'path'
+CACHEFILE = 'favicon.ico'
+
+module.exports = (req, res, next) ->
+	res.sendFile CACHEFILE,
+		root: tmpdir
+		(error) ->
+			if error
```

Comment:
Since there's only one item in the object you're passing as options, I think it's neater to have it on the same line, and the signature of the callback too. It saves you a level of indentation, and looks clearer (at least to my eye). So you'd have this:
```
res.sendFile CACHEFILE, root: os.tmpdir(), (error) ->
	if error
```